### ISSUE TITLE
relativeCSSInjection: true will not inject css code

### ISSUE THREAD

**Issue by tjx666** [marco-prontera/vite-plugin-css-injected-by-js#84]
check here: https://github.com/tjx666/vite-plugin-css-injected-by-js-issue/tree/947eff9f8707b606a8873d571c0dc914cb600183

![image](https://user-images.githubusercontent.com/41773861/217988121-6a19c5c2-b14c-42e0-a6e6-bf93bc2c4f60.png)

Versions:

```
System:
    OS: macOS 13.2
    CPU: (8) x64 Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz
    Memory: 1.15 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.14.0 - ~/Library/Caches/fnm_multishells/6278_1675997414513/bin/node
    Yarn: 1.22.19 - ~/Library/Caches/fnm_multishells/6278_1675997414513/bin/yarn
    npm: 9.4.2 - ~/Library/Caches/fnm_multishells/6278_1675997414513/bin/npm
  npmPackages:
    @vitejs/plugin-vue: ^4.0.0 => 4.0.0 
    @vitejs/plugin-vue2: ^2.2.0 => 2.2.0 
    @vitejs/plugin-vue2-jsx: ^1.1.0 => 1.1.0 
    vite: ^4.0.4 => 4.0.4 
    vite-plugin-css-injected-by-js: ^3.0.0 => 3.0.0 
    vite-plugin-vue-type-imports: ^0.2.4 => 0.2.4 
```

**Comment by marco-prontera** [comment]
Thank you for the issue, but why import CSS that isn't actually used?

In the event that it is used, Vite will already take care of it, as in the following case:
```
import styles from './a.less';

console.log(123, styles.app);
```

The output of the build will be:
```
"use strict";
const styles = ".app {\n  color: red;\n}\n";
console.log(123, styles.app);
```

I can't understand the need, can you explain the reason and what is the purpose?

Since the building process doesn't give information about the imported CSS by the JS file, we can't inject the CSS into its importer. Maybe I can help you better if you explain to me what you expect

**Comment by tjx666** [comment]
When css file is imported, it had been used.
css file born with effect.

**Comment by marco-prontera** [comment]
> When css file is imported, it had been used.
> 
> css file born with effect.

Yes you are right, but the usage was not clear to me since I always use classes directly. But I figured out this need, so I'm fixing it, thanks!

**Comment by tjx666** [comment]
import css directly is a very normal use case, for example, import the css variables, like my company project:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/41773861/218819325-f50b4b21-77dd-4632-b1ed-5e037291e1b7.png">

**Comment by marco-prontera** [comment]
Of course, I was referring to the act of having split entry points. anyway don't worry, tomorrow release with the fix 😜

**Comment by marco-prontera** [comment]
@tjx666 fixed in [3.0.1](https://www.npmjs.com/package/vite-plugin-css-injected-by-js/v/3.0.1) 
Unfortunately, it's a period where I'm busy, so if you want to help fix possible problems it's welcome ;)

**Comment by tjx666** [comment]
Seems latest version 3.0.1 relativeCSSInjection: true option doesn't work.
